templates/form/input--submit.html.twig

```diff
@@ -13,6 +13,6 @@
 
 {% include '@ecl-twig/button' with {
   label: attributes.value,
-  variant: 'primary',
+  variant: attributes.variant|default('primary'),
```

Comment:
Mmmm, not really good to use the attributes array for this. That will also get printed onto the element in html.

Just make a separate variable just for that.